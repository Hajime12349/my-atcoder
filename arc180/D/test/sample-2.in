10 15
8 3 8 10 1 5 3 1 6 4
4 6
2 5
6 9
8 10
2 9
4 10
1 5
1 8
1 3
4 8
1 10
2 10
6 10
2 6
2 6
