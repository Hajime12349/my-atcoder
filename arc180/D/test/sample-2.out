16
14
12
11
17
17
19
14
19
14
17
17
12
16
16
